docs/ops/type/Convert_1.md

```diff
@@ -4,7 +4,16 @@
 
 **Category**: type conversion
 
-**Short description**: Operation converts all elements of the input tensor to a type specified in the *"destination_type"* attribute.
+**Short description**: Operation converts all elements of the input tensor to a type specified in the *destination_type* attribute.
```

Comment:
*Convert* operation performs element-wise conversion, on a given input tensor, to a type...

Reply:
Done [here](https://github.com/openvinotoolkit/openvino/pull/5530/commits/3921c0f91cf899ed0c2577ddaf0b5841159669c8)